mcs/class/referencesource/System.Data/System/Data/Common/DbDataAdapter.cs

```diff
@@ -21,8 +21,8 @@ namespace System.Data.Common {
     public abstract class DbDataAdapter : DataAdapter, IDbDataAdapter, ICloneable { // V1.0.3300, MDAC 69629
         public const string DefaultSourceTableName = "Table"; // V1.0.3300
 
-        internal static readonly object ParameterValueNonNullValue = 0;
-        internal static readonly object ParameterValueNullValue = 1;
+        internal static readonly object s_parameterValueNonNullValue = 0;
```

Comment:
It was an intermediate commit to compile build, now this file is not used at all - I am going to remove the whole mcs/class/referencesource/System.Data/ folder once I finish or I can remove files with each change..